core/templates/dev/head/App.js

```diff
@@ -24,7 +24,7 @@ var oppia = angular.module(
     'angularAudioRecorder', 'dndLists', 'headroom', 'infinite-scroll',
     'ngAnimate', 'ngAudio', 'ngCookies', 'ngImgCrop', 'ngJoyRide', 'ngMaterial',
     'ngResource', 'ngSanitize', 'ngTouch', 'pascalprecht.translate', 'toastr',
-    'ui.bootstrap', 'ui.sortable', 'ui.tree', 'ui.validate'
+    'ui.bootstrap', 'ui.sortable', 'ui.tree', 'ui.validate', 'alertMessageModule'
```

Comment:
Should this be alphabetized?

Reply:
I am not sure about that. Although alphabetizing seems lucrative, but blindly sorting the dependencies just according to their alphabets would be bizarre. What we can do is have a sorting similar to our current [sorted dependency check](https://github.com/oppia/oppia/blob/bff3e006daebb26e2390b4c7560896fe521615ba/scripts/pre_commit_linter.py#L1323) and have all the module imports together and alphabetically sorted among themselves.
What do you think?

Reply:
I'm fine with anything as long as there's a clearly-documented system, so that other devs know what to do.

Reply:
Oh okay I'll comments regarding this and also update it in the style guide.